corehq/apps/sms/tests/test_views.py

```diff
@@ -131,7 +131,7 @@ def domain_admin():
 @contextmanager
 def contractor():
     with domain_admin() as user, flag_enabled('IS_CONTRACTOR'):
-        yield
+        yield user
```

Comment:
I'm curious about the purpose of this. I can see a reason for doing it if you had code that used it, but all of the places that use this context manager ignore the context object (`user`).

Reply:
:rofl: You are absolutely right! How did I not notice that?!

I wrote the context managers first, and I think assumed I'd use `user` for `client.login()`, and then didn't.